config/routes.rb

```diff
@@ -7,6 +7,9 @@
 
 Discourse::Application.routes.draw do
   mount ::DiscourseGamification::Engine, at: "/leaderboard"
+  get "/admin/plugins/discourse-gamification" =>
+        "discourse_gamification/admin_gamification_leaderboard#index",
+      :constraints => StaffConstraint.new
   get "/admin/plugins/gamification" =>
```

Comment:
Did you intend to keep this route?

Reply:
Yeah, since the new single plugin page (and the gamification JS route) will now be `admin/plugins/discourse-gamification` . I may have to remove this in a later PR, not sure yet. But for this changeover period I need it.

Reply:
Actually you are right lol I didn't need this :D